belvedere/src/main/java/zendesk/belvedere/FloatingActionMenu.java

```diff
@@ -136,15 +139,31 @@ private void rotate(boolean isExpanded) {
         ViewCompat.animate(fab).rotation(angle).setDuration(animationDuration).start();
     }
 
-    public void addMenuItem(@DrawableRes int iconId, @NonNull View.OnClickListener clickListener) {
-        FloatingActionButton fab = (FloatingActionButton) layoutInflater.inflate(R.layout.floating_action_menu_item, this, false);
-        fab.setOnClickListener(clickListener);
+    public void addMenuItem(@DrawableRes int iconRes, @NonNull View.OnClickListener clickListener) {
+        FloatingActionButton menuItem = (FloatingActionButton) layoutInflater.inflate(R.layout.floating_action_menu_item, this, false);
 
-        fab.setBackgroundResource(iconId);
-        fab.setBackgroundTintList(ColorStateList.valueOf(ContextCompat.getColor(getContext(), R.color.floating_action_menu_item_background)));
+        // Reference: https://stackoverflow.com/a/30688147
```

Comment:
The accepted answer is to use `useCompatPadding`. Why can we not do that?

https://stackoverflow.com/a/30879360